requirements/quality.txt

```diff
@@ -32,7 +33,7 @@ enum34==1.1.6
 fs==2.4.0
 funcsigs==1.0.2           # via mock
 future==0.17.1            # via backports.os
-futures==3.2.0            # via caniusepython3, isort
+futures==3.2.0; python_version < "3.0"  # via caniusepython3, isort
```

Comment:
There should instead by a `requirements/constraints.txt` file similar to the one in https://github.com/edx/event-tracking/pull/50 (plus the `-c constraints.txt` line in each `requirements/*.in` file).  This is a pattern we discovered after [OEP-18](https://open-edx-proposals.readthedocs.io/en/latest/oep-0018-bp-python-dependencies.html) was written, I plan to make an update soon to mention it.